backend/api/Migrations/20241210115843_AddInspectionTagMetadataTable.cs

```diff
@@ -14,41 +14,51 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 name: "IsarZoomDescription_ObjectHeight",
                 table: "MissionTasks",
                 type: "double precision",
-                nullable: true);
```

Comment:
Should migrations be excluded from this check?

Reply:
Yes, absolutely, fixed now =)